.github/workflows/build_and_test_dbt.yaml

```diff
@@ -31,15 +41,20 @@ jobs:
           path: ${{ env.PROJECT_DIR }}/${{ env.STATE_DIR }}
           key: ${{ env.CACHE_KEY }}
 
-      - if: steps.cache.outputs.cache-hit == 'true'
-        name: Set command args to build/test modified resources
-        run: echo "MODIFIED_RESOURCES_ONLY=true" >> "$GITHUB_ENV"
-        shell: bash
-
-      - if: steps.cache.outputs.cache-hit != 'true'
-        name: Set command args to build/test all resources
-        run: echo "MODIFIED_RESOURCES_ONLY=false" >> "$GITHUB_ENV"
+      - name: Set command args for building/testing resources
+        run: |
+          if [[ $CACHE_HIT == 'true' ]]; then
+            echo "MODIFIED_RESOURCES_ONLY=true" >> "$GITHUB_ENV"
+          else
+            echo "MODIFIED_RESOURCES_ONLY=false" >> "$GITHUB_ENV"
+          fi
+          if [[ $EVENT_NAME == 'workflow_dispatch' ]]; then
+              echo "MANUALLY_DISPATCHED=true" >> "$GITHUB_ENV"
+          fi
```

Comment:
**[Thought, non-blocking]** The lack of an `else` branch where we set `MANUALLY_DISPATCHED=false` seems like it doesn't mess up the check on line 67, probably because these bash blocks inherit the default behavior of continuing execution on unset variables; this makes me wonder whether the corresponding branch to set `MODIFIED_RESOURCES=false` on lines 48-50 above is also unnecessary 🤔 But that's not the focus of this PR so I'm fine punting on it until later!

Reply:
Fixed in [164b58a](https://github.com/ccao-data/data-architecture/pull/142/commits/164b58a00d3e02ea1090269d22469109f03b4d30)!